lib/screens/explore_screens/vouch/vouched_tab/vouched_tab.dart

```diff
@@ -31,7 +32,16 @@ class VouchedTab extends StatelessWidget {
                   return const VouchSuccessDialog();
                 },
               );
+            } else if (pageCommand is ShowNotQualifiedToVouch) {
+              showDialog<void>(
+                context: context,
+                barrierDismissible: false,
+                builder: (_) {
+                  return const NotQualifiedToVouchDialog();
+                },
+              );
             }
+
             BlocProvider.of<VouchedBloc>(context).add(const ClearPageCommand());
           },
           builder: (context, VouchedState state) {
```

Comment:
not needed specify the state type
` builder: (context,  state)`

Reply:
it doesnt hurt to have it. I like having typed variables over not typed

Reply:
I would advocate here for "consistency" since it would be the first place in the entire project where the state type is specified.

Also you can easily know the type as it is defined and in the signature of BlocBuilder / BlocConsumer